spec/truffle/array/shuffle_spec.rb

```diff
@@ -0,0 +1,7 @@
+require_relative '../../ruby/spec_helper'
+
+describe "Array#shuffle" do
+  it "matches cruby" do
+    %w[a b c].shuffle(random: Random.new(1)).should == %w[a c b]
```

Comment:
Could you move this to `spec/ruby/core/array/shuffle_spec.rb`?
Also could you add more examples than just this one? For instance one with 10+ elements.

Also it would be good to test the same but without the `random:` kwarg, and using `srand` instead to fix the seed.